block-filter/src/filter.rs

```diff
@@ -3,6 +3,7 @@ use ckb_logger::debug;
 use ckb_notify::NotifyController;
 use ckb_store::{ChainDB, ChainStore};
 use ckb_types::{core::HeaderView, prelude::*};
+use golomb_coded_set::{GCSFilterWriter, SipHasher24Builder, M, P};
 use std::thread;
 
 const THREAD_NAME: &str = "ckb-block-filter-thread";
```

Comment:
Only 16 bytes are supported for a thread name, and the last byte should be `NULL`.

Since a long thread name will cut to 15 characters, maybe `CamelCase` is better than `snake_case`.

Ref: This limit is created by kernel.

```c
/* Task command name length: */
#define TASK_COMM_LEN     16
```

Ref: [The definition of `TASK_COMM_LEN` in latest stable kernel.](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/include/linux/sched.h?h=v5.10.12#n213)